packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -144,7 +144,7 @@ export default connectStore(() => {
         <Col mediumSize={3}>
           <BlockLink to={`/vms/${id}/disks`}>
             <h2>
-              {formatSize(vmTotalDiskSpace)} <Icon icon='disk' size='lg' />
+              {formatSizeShort(vmTotalDiskSpace)} <Icon icon='disk' size='lg' />
```

Comment:
@pdonias Do you agree with this change?

Reply:
So "there is 1 decimal between -10 and 10 excluded and none out of this interval", right? If so, I think this should work fine, yes.

Reply:
Exactly.